js/index.js

```diff
@@ -1,19 +1,28 @@
-// const addAnswerButton = document.querySelector(
-//   '[data-js="show-answer-button"]'
-// );
-// const addHiddenAnswer = document.querySelector('[data-js="hidden-answer"]');
-// const addBookmark = document.querySelector('[data-js="bookmark-button"]');
+const AnswerButton = document.querySelector('[data-js="show-answer-button"]');
+const addAnswer = document.querySelector('[data-js="hidden-answer"]');
+const Bookmark = document.querySelector('[data-js="bookmark"]');
 
-// addAnswerButton.addEventListener("click", () => {
-//   addHiddenAnswer.classList.toggle("answer");
-// });
-
-const darkMode = document.querySelector('[data-js="checkbox"]');
+AnswerButton.addEventListener("click", function handleClick() {
+  addAnswer.classList.toggle("card__answer");
 
-darkMode.addEventListener("click", () => {
-  if (document.body.classList.contains("body-dark")) {
-    document.body.classList.remove("body-dark");
+  const initialText = "Show answer";
+  if (AnswerButton.textContent.includes(initialText)) {
+    AnswerButton.textContent = "Hide answer";
   } else {
-    document.body.classList.add("body-dark");
+    AnswerButton.textContent = initialText;
   }
 });
+
+Bookmark.addEventListener("click", () => {
+  Bookmark.classList.toggle("card__bookmarked");
+});
+
+// const darkMode = document.querySelector('[data-js="checkbox"]');
+
+// darkMode?.addEventListener("click", () => {
+//   if (document.body.classList.contains("body-dark")) {
+//     document.body.classList.remove("body-dark");
+//   } else {
+//     document.body.classList.add("body-dark");
+//   }
+// });
```

Comment:
We normally don't want to merge this kind of lines; consider to remove them. Next time, you'll write the code even better ;)